### ISSUE TITLE
Issue with example's command

### ISSUE THREAD

**Issue by CyrilKrylatov** [HugoGiraudel/node-legofy#26]
Hello guys,

Seems like I have a little problem with the main command: `$ legofy < photo.jpg > photo.lego.jpg`

I tried with this following photo: http://ss.c-krylatov.com/RCw0fRbjX5.jpg (yes, I glitched it)

It created a `photo.lego.jpg` but the file is empty (0 octet)

Here is the stack trace:

```
$ legofy < photo.jpg > photo.lego.jpg
/usr/local/lib/node_modules/node-legofy/dist/index.js:60
    var _ctx$getImageData = ctx.getImageData(x, y, brick.width, brick.height);
                                ^

Error: IndexSizeError: The source width is 0.
    at Error (native)
    at /usr/local/lib/node_modules/node-legofy/dist/index.js:60:33
    at render (/usr/local/lib/node_modules/node-legofy/dist/index.js:117:7)
    at /usr/local/lib/node_modules/node-legofy/dist/index.js:130:12
    at ConcatStream.<anonymous> (/usr/local/lib/node_modules/node-legofy/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:67:13)
    at ConcatStream.emit (events.js:166:7)
    at finishMaybe (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:509:14)
    at endWritable (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:519:3)
    at ConcatStream.Writable.end (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:484:5)
```

Thanks!

**Comment by KittyGiraudel** [comment]
```
bin/legofy < /Users/hgiraudel/Desktop/RCw0fRbjX5.jpg > test.lego.jpg
Image LEGOfied in 0.068 seconds.
```

![test lego](https://cloud.githubusercontent.com/assets/1889710/11067473/fbf0c422-87cc-11e5-875b-7a586ed66db7.jpg)

**Comment by valeriangalliat** [comment]
Same on my side, was about to post the same output.

What Node.js version are you using?

**Comment by CyrilKrylatov** [comment]
@valeriangalliat 
`$ node -v
v4.1.1`

**Comment by valeriangalliat** [comment]
Edit: I can reproduce regardless the version with the globally installed version while it's okay with `bin/legofy`. Might have been a problem with `npm publish`.

**Comment by KittyGiraudel** [comment]
I know for a fact it works in both `0.12.x` and `5.0.0`. Let me check on `4.1.1`.

**Comment by KittyGiraudel** [comment]
@valeriangalliat Oh, so it's node a Node issue. More like the global command that doesn't work.

**Comment by valeriangalliat** [comment]
Fuck we didn't publish `images/brick.png`

**Comment by Dakuan** [comment]
I also have issues with the global

```
 -> legofy < ~/Desktop/xp.jpg > test.lego.jpg
/usr/local/lib/node_modules/node-legofy/dist/index.js:113
  ctx.drawImage(image, 0, 0);
      ^

Error: Image given has not completed loading
    at Error (native)
    at render (/usr/local/lib/node_modules/node-legofy/dist/index.js:113:7)
    at /usr/local/lib/node_modules/node-legofy/dist/index.js:130:12
    at ConcatStream.<anonymous> (/usr/local/lib/node_modules/node-legofy/node_modules/concat-stream/index.js:36:43)
    at emitNone (events.js:67:13)
    at ConcatStream.emit (events.js:166:7)
    at finishMaybe (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:509:14)
    at endWritable (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:519:3)
    at ConcatStream.Writable.end (/usr/local/lib/node_modules/node-legofy/node_modules/readable-stream/lib/_stream_writable.js:484:5)
    at Stream.method [as end] (/usr/local/lib/node_modules/node-legofy/node_modules/duplexer/index.js:47:39)
```

**Comment by Dakuan** [comment]
@valeriangalliat 

![image](https://cloud.githubusercontent.com/assets/633621/11067637/7bd62d60-87c5-11e5-8d04-dc5c84fe2774.png)

**Comment by valeriangalliat** [comment]
Fixed as of 0.1.5! Thanks for reporting.

**Comment by CyrilKrylatov** [comment]
Yay! thx <3
